ray_lightning/ray_horovod.py

```diff
@@ -237,3 +195,16 @@ def root_device(self):
                 return torch.device("cuda", 0)
         else:
             return torch.device("cpu")
+
+    def set_cuda_device_if_used(self):
```

Comment:
Should this method be defined in the _GPUAccelerator instead of in this strategy?